synchronizer/models.py

```diff
@@ -9,7 +9,7 @@
 
 from synchronizer.connectors.base import ExportException
 from synchronizer.connectors.manager import ConnectorManager
-from synchronizer.utils import AESCipher, DateAndTime
+from synchronizer.utils import AESCipher, DateAndTime, validate_connector
```

Comment:
Each connector should have its own `validate_connector` method because the logic will be different